### PR TITLE
Added points to clarify how PoW achieves consensus

### PR DESCRIPTION
**Pull request by YashKarthik**
## Description

Previously, the page explained how PoW works etc, but didn't explain the _connection_ between PoW and consensus.

I added a couple of bullets that briefly explain that connection and the fact that PoW isn't there to _prove_ something but to _deter_ manipulation of blocks.

### CODE REVIEW

File: src/content/developers/docs/consensus-mechanisms/pow/index.md [hunk 1]
```diff
@@ -20,8 +20,17 @@ Proof-of-work is the mechanism that allows the decentralized Ethereum network to
...
+Decentralized consensus requires two things:
+
+- A common objective: extending the chain (motivated by economic incentives)
+- A way to measure our efforts toward the objective: proof of work
+
 Proof-of-work is the underlying algorithm that sets the difficulty and rules for the work miners do. Mining is the "work" itself. It's the act of adding valid blocks to the chain. This is important because the chain's length helps the network follow the correct Ethereum chain and understand Ethereum's current state. The more "work" done, the longer the chain, and the higher the block number, the more certain the network can be of the current state of things.
 
+**NOTE:**
+
+The purpose of proof of work algorithms is *not* proving that certain work was carried out, but deterring manipulation of data by establishing large energy and hardware requirements to be able to do so. [[51% Attacks]](https://en.wikipedia.org/wiki/Double-spending#51%_attack)
```
Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/pow/index.md:32]: IMO this section is already covered under the 'Proof-of-work and security' heading

> To consistently create malicious yet valid blocks, you'd need over 51% of the network mining power to beat everyone else. You'd need a lot of computing power to be able to do this amount of "work". And the energy spent might even outweigh the gains you'd make in an attack.

File: src/content/developers/docs/consensus-mechanisms/pow/index.md [hunk 2]
```diff
@@ -20,8 +20,17 @@ Proof-of-work is the mechanism that allows the decentralized Ethereum network to
 
 ## Proof-of-work and mining {#pow-and-mining}
 
+Decentralized consensus requires two things:
+
+- A common objective: extending the chain (motivated by economic incentives)
```
Comment (minimalsm) [inline src/content/developers/docs/consensus-mechanisms/pow/index.md:25]: Same as the other section, I think we already cover this adequately on this page or in the prerequsities for this page.

**Comment by minimalsm** [comment]
Thanks again for the PR and discussion here. Going to close this out as per my comments above

@all-contributors please add @YashKarthik for ideas

**Comment by YashKarthik** [comment]
No problem :)

The wiki helped me a lot to understand ethereum, I'll look around and see where I can help.

Cheers

**Pull request closed without merging**